src/ButtonPanel.ts

```diff
@@ -1,14 +1,21 @@
 /* The ButtonPanel component */
 import './style.css'
 import log from 'loglevel'
+//@ts-expect-error No need for declaration files for images yet
 import leftArrowComponent from './images/left-arrow.svg'
+//@ts-expect-error No need for declaration files for images yet
 import rightArrowComponent from './images/right-arrow.svg'
+import { ButtonPanelMethods } from './types'
 
-export default class ButtonPanel {
+export default class ButtonPanel implements ButtonPanelMethods {
   leftButtonId = 'next-left-arrow'
   rightButtonId = 'next-right-arrow'
+  buttonPanel: HTMLDivElement
+  leftArrow: HTMLDivElement
+  rightArrow: HTMLDivElement
+  private _isHidden: Boolean
```

Comment:
`Boolean` is an object type `boolean` is an primitive type, we should always try to use the primitive type. If you want more in depth information checkout the [everyday types](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html) page of the TypeScript Lang webiste